home.html

```diff
@@ -71,6 +72,7 @@ <h3 class="section-header">Why you should Start Coding?</h3>
                 <button class="btn">Read more <span><i class="ri-arrow-right-up-line"></i></span></button>
             </p>
         </div>
+        <!--Top Stories Section-->
```

Comment:
The background-color of latest articles is visible at the extreme right section of the top stories section. Fix it.